src/bsp/platform/stm32fx_nucleo_wifi/xi_bsp_tls_certs.h

```diff
@@ -0,0 +1,101 @@
+#define GLOBALSIGN_ROOT_CERT "-----BEGIN CERTIFICATE-----\n\
```

Comment:
yeah, probably JC took this from xi_RootCA_list.c, right JC?

1. in file xi_RootCA_list.c these are concatenated together into a single array since wolfSSL is able to handle multiple Root CAs. Here we could try whether the wifi module handles multiple TLS certs passed to API, could you test this JC please?

2. Does not double the storage for two reasons: a) the xi_RootCA_list array is not compiled in here, because no TLS layer is compiled in either because the TLS is done by the wifi chip. b) is my guess that even if it it was included the compiler would drop it out if it isn't used, but this is up to the compiler.

3. Do you know why we initialize the xi_RootCA_list with hexa numbers? I mean we could initialize the array with human-readable string, with the same string JC uses here: PEM format cert.